src/test/java/com/jcabi/aether/RootArtifactTest.java

```diff
@@ -86,6 +91,32 @@ public void resolvesRootArtifact() throws Exception {
     }
 
     /**
+     * RootArtifact can resolve a parent artifact.
```

Comment:
@prondzyn sorry for the delay ... was a little sick with the flu or something :) 

That being said. I was just asking for explicitly stating.

> RootArtifact can resolve a parent artifact. In this example the parent of `jcabi-aether` version `0.10.1` is correctly resolved  to `com.jcabi.jar` v. ...

Just so that the reader of this doesn't need explicit understanding of what's going on with the versions here :)